cime_config/tests.py

```diff
@@ -402,34 +400,34 @@
 
     "eam_nl" : {
         "tests"    : (
-            "SBN.ne4_ne4.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne11_ne11.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne16_ne16.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne30_ne30.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne45pg2_r05_oECv3.FC5AV1C-L.allactive-thetadycore",
-            "SBN.ne120_ne120.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne240_ne240.FC5AV1C-L.eam-thetadycore",
-            "SBN.ne512np4_360x720cru_ne512np4.FC5AV1C-L.allactive-thetadycore",
-            "SBN.ne1024np4_360x720cru_ne1024np4.FC5AV1C-L.allactive-thetadycore",
-            "SBN.conusx4v1_conusx4v1.FC5AV1C-L.eam-thetadycore",
-            "SBN.enax4v1_enax4v1.FC5AV1C-L.eam-thetadycore",
-            "SBN.northamericax4v1_r0125_northamericax4v1.FC5AV1C-L.eam-thetadycore",
-            "SBN.antarcticax4v1_r0125_antarcticax4v1.FC5AV1C-L.allactive-thetadycore",
-            "SBN.antarcticax4v1pg2_r0125_antarcticax4v1pg2.FC5AV1C-L.allactive-thetadycore",
-            "SBN.ne4_ne4.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne11_ne11.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne16_ne16.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne30_ne30.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne45pg2_r05_oECv3.FC5AV1C-L.allactive-preqxdycore",
-            "SBN.ne120_ne120.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne240_ne240.FC5AV1C-L.eam-preqxdycore",
-            "SBN.ne512np4_360x720cru_ne512np4.FC5AV1C-L.allactive-preqxdycore",
-            "SBN.ne1024np4_360x720cru_ne1024np4.FC5AV1C-L.allactive-preqxdycore",
-            "SBN.conusx4v1_conusx4v1.FC5AV1C-L.eam-preqxdycore",
-            "SBN.enax4v1_enax4v1.FC5AV1C-L.eam-preqxdycore",
-            "SBN.northamericax4v1_r0125_northamericax4v1.FC5AV1C-L.eam-preqxdycore",
-            "SBN.antarcticax4v1_r0125_antarcticax4v1.FC5AV1C-L.allactive-preqxdycore",
-            "SBN.antarcticax4v1pg2_r0125_antarcticax4v1pg2.FC5AV1C-L.allactive-preqxdycore",
+            "SBN.ne4_oQU240.F2010.eam-thetadycore",
+            "SBN.ne11_ne11.F2010-CICE.eam-thetadycore",
```

Comment:
For this set of tests, not all the grids are ready for F2010 with mpassi for prescribed ice. Only ne4_ne4 that is also replaced for the integration tests is using the new grid to go with F2010. F2010-CICE is used for the other grids, as these tests are just for eam_nl.